app/controllers/errors_controller.rb

```diff
@@ -1,6 +1,5 @@
-class ErrorsController < ApplicationController
-  skip_before_action :verify_authenticity_token,
-                     :request_login
+class ErrorsController < ActionController::Base
```

Comment:
Feels like this particular change to the ErrorsController deserves a separate commit?

Reply:
I'm not familiar with why this change was included, I think @asmega will know more than me here

Reply:
We can change back to inheriting from `ApplicationController` for the sake of this PR we will just need to add the additional skip filters

Reply:
It is a bit of a pain to switch it back, the reason it was changed seems to be that all of the default Application Controller behaviour isn't needed if you're showing an error page - it skips all of it anyway, and adding in this as an extra skip doesn't make it any more useful. I do think it could be a separate commit though, seeing as it's got quite a broad effect?

Reply:
This now inherits from `ApplicationController`